### ISSUE TITLE
Dex Hunter widget

### ISSUE THREAD

**Issue by willbasky** [encryptedcoins/encoins-frontend#141]
a direct swap window to Dex Hunter.
a widget for integration with Dex Hunter to tasks for the front. This is on the main page.

The 2nd one is like a widget:
```html
<!-- ADD TO <head> -->
        <script src="https://unpkg.com/react@18.2/umd/react.production.min.js" crossorigin></script>
        <script src="https://unpkg.com/react-dom@18.2/umd/react-dom.production.min.js" crossorigin></script>
        <script type="module" src="https://unpkg.com/@dexhunterio/swaps@0.0.86/lib/umd/swaps.umd.js"></script>
    
        <!-- ADD TO <body> -->
        <div id="dexhunter-root"></div>
        <script type="module">
            ReactDOM.render(
                React.createElement(
                    dexhunterSwap, 
                    {"orderTypes":["SWAP","LIMIT"],"colors":{"background":"#0E0F12","containers":"#191B23","subText":"#88919E","mainText":"#FFFFFF","buttonText":"#FFFFFF","accent":"#007DFF"},"theme":"dark","width":450,"partnerCode":"encoins61646472317138753272683575756436797a6d687130646537767437703072767170666164776e656533746a6e7a32746c347263743671743033776a63326c667779716e6435346777666465793530733733343265336a6c366b78777777346b717a6661683278da39a3ee5e6b4b0d3255bfef95601890afd80709","partnerName":"Encoins"}
                ),
                document.getElementById('dexhunter-root')
            );
        </script>
```

The 3rd one I guess is not necessary but I paste it here just in case
```html
<Swap
     orderTypes={["SWAP","LIMIT"]}
     colors={{"background":"#0E0F12","containers":"#191B23","subText":"#88919E","mainText":"#FFFFFF","buttonText":"#FFFFFF","accent":"#007DFF"}}
     theme="dark"
     width="450"
     partnerCode="encoins61646472317138753272683575756436797a6d687130646537767437703072767170666164776e656533746a6e7a32746c347263743671743033776a63326c667779716e6435346777666465793530733733343265336a6c366b78777777346b717a6661683278da39a3ee5e6b4b0d3255bfef95601890afd80709"
     partnerName="Encoins"
     displayType="FULL"
   />
```

**Comment by willbasky** [comment]
On main page under introduction

**Comment by willbasky** [comment]
> On main page under introduction

Fixed in #180
